.gitignore

```diff
@@ -53,6 +53,7 @@ __pycache__/
 # C extensions
 *.so
 *.c
+*.cpp
```

Comment:
snuck this in b/c git kept wanting to add oasis.cpp

Reply:
We'll need to remember this if we ever add real cpp files to the project, but this is good for now.

Reply:
put the full path to `oasis.cpp`?

Reply:
If we're doing that, since this is the only C extension, we could also remove `*.c` for consistency?

Reply:
The full path is a good idea. How much we use wildcards isn't something I care much about; I suspect the initial form of that file probably came from a template someone grabbed way back.

Reply:
Sorry I'm a little confused - what should I update this to exactly?

Reply:
It's fine to remove the *.c if you want, and to refactor the cpp filter to be only pathed rather than general. Up to you.